bucket/vortex.json

```
@@ -1,10 +1,10 @@
 {
     "homepage": "https://www.nexusmods.com/about/vortex/",
     "description": "Open-source mod manager from Nexus Mods",
-    "version": "1.1.14",
+    "version": "1.2.10",
     "license": "GPL-3.0-or-later",
-    "url": "https://github.com/Nexus-Mods/Vortex/releases/download/v1.1.14/vortex-setup-1.1.14.exe#/dl.7z",
-    "hash": "sha256:3db03248afc610e4cc39950888db15e86811631b76a5ab5a4aa694ea142d5893",
+    "url": "https://github.com/Nexus-Mods/Vortex/releases/download/v1.2.10/vortex-setup-1.2.10.exe#/dl.7z",
+    "hash": "sha256:51a3bd071b3218d2cc7c2ff0a16262d46def5d254868b79a810d4fbb96f579e7",
```

```suggestion
    "hash": "ce38d714ac0dda0fa4d7cf21df13678b3d6531c852d99e07f881507e5efe62df",
```

The redundant `sha256:` prefix can also be dropped.